src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy

```diff
@@ -46,7 +47,7 @@ class GCloudConfigurable {
     }
 
     void args(Map<String, ?> args) {
```

Comment:
Yeah, so when doing map1 + map2 using Map<String, ?> (https://github.com/GoogleCloudPlatform/gradle-appengine-plugin/blob/master/src/main/groovy/com/google/gcloud/GCloudPlugin.groovy#L91) is an unchecked warning, this gets rid of that by ensuring the maps are compatible.